app/controllers/services_controller.rb

```diff
@@ -115,4 +115,14 @@ def ask_the_eu_link
     %q(<a href="http://asktheeu.org">Ask The EU</a>)
   end
 
+  def request_link(request_path)
```

Comment:
👎 – pretty sure the following would fix this:

```diff
diff --git a/app/controllers/services_controller.rb b/app/controllers/services_controller.rb
index 079f69d7f..76771a6c0 100644
--- a/app/controllers/services_controller.rb
+++ b/app/controllers/services_controller.rb
@@ -61,7 +61,7 @@ class ServicesController < ApplicationController
            locals: {
              name_to: info_request.user_name.html_safe,
              info_request: info_request, :reason => params[:reason],
-             info_request_url: request_link(request_path(info_request)),
+             info_request_url: request_url(info_request),
              site_name: site_name.html_safe }
   end
```

The specs do fail, but that's because `config.force_ssl` is  set in the environment initialiser so can't be stubbed.

Its worth checking that I'm not talking rubbish here, but I'm pretty sure the helpers take protocol in to account…